### ISSUE TITLE
could not compile smallvec

### ISSUE THREAD

**Issue by aeosynth** [fleabitdev/glsp#4]
```
> cargo r
   Compiling smallvec v1.4.0
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> /home/james/.cargo/registry/src/github.com-1ecc6299db9ec823/smallvec-1.4.0/lib.rs:61:32
   |
61 | #![cfg_attr(feature = "union", feature(untagged_unions))]
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0554`.
error: could not compile `smallvec`.
```

Void Linux
cargo new, glsp = "0.1"

**Comment by fleabitdev** [comment]
Thanks for the report!

GameLisp currently requires `nightly` Rust - are you using the `stable` release channel?

**Comment by aeosynth** [comment]
Yes I'm on stable. I missed that you mentioned it in the overview, maybe mention it in the readme as well? Closing.

**Comment by fleabitdev** [comment]
Good idea - it's easily missed. Fixed by 30f7f8f.
